src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(6, 2)).toEqual([3, 3]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(8, 1)).toEqual([8]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(17, 4)).toEqual([4, 4, 4, 5]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
-
+  expect(splitInteger(32, 6)).toEqual([5, 5, 5, 5, 6, 6]);
```

Comment:
The test case description suggests that zeros should be added if the value is less than the number of parts. However, the expected result `[5, 5, 5, 5, 6, 6]` does not include zeros. According to the task description, the array should contain exactly `numberOfParts` elements, and the difference between the max and min should be <= 1. Consider revising the expected result to include zeros if necessary.